src/main/java/com/iexec/commons/poco/chain/adapter/CommandStatus.java

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Shouldn't this package be in blockchain-adapter library?

Reply:
it totally could be done like this.
What about doing it in a next step while deciding if endpoints related to `contribute` and `reveal` should be removed from blockchain-adapter as well ?